components/dash-core-components/src/fragments/RangeSlider.react.js

```diff
@@ -101,8 +103,9 @@ export default class RangeSlider extends Component {
                         getTooltipContainer: node => node,
                     }}
                     style={{position: 'relative'}}
-                    value={value}
-                    marks={truncatedMarks}
+                    value={calcValue(min, max, value)}
+                    marks={calcMarks({min, max, marks, step})}
```

Comment:
will want to skip this if `marks` is supplied by the user

Reply:
`calcMarks` provides truncation only when it's provided, but we don't want any validation when supplied by user?